jaeger_client/rate_limiter.py

```diff
@@ -1,22 +1,16 @@
-# Copyright (c) 2017 Uber Technologies, Inc.
+# Copyright (c) 2016 Uber Technologies, Inc.
```

Comment:
looks like 2017 was correct on this file

Reply:
Fixed - and on test.